docs/configuration/README.md

```diff
@@ -70,7 +70,7 @@ When you run EventStoreDB with this option, it will print out the effective conf
 ```
 $ eventstore --what-if
 [1085525, 1,09:19:40.884,INF]
-"ES VERSION:"             "21.2.1.0" ("configuration"/"10b6de53a", "Unknown")
+"ES VERSION:"             "21.6.0.0" ("configuration"/"10b6de53a", "Unknown")
```

Comment:
**SUGGESTION:** It would be good to update with the proper hash once the 21.6 is released.